packages/react/.storybook/preview.tsx

```diff
@@ -42,7 +42,7 @@ const preview: Preview = {
         },
         options: {
             storySort: (a, b) => {
-                const roots = ['Getting Started', 'Components', 'Migrating'];
+                const roots = ['Getting Started', 'Components', 'Migrating', 'Misc'];
```

Comment:
Is it required to add `Misc` stories in the storybook UI ?

Reply:
Yep otherwise it is rendered at the top of the sidebar

Reply:
No way to hide it ?

Reply:
Nope I searched a bit :/ 

https://github.com/storybookjs/storybook/issues/17801

Reply:
I just stumbled across your PR here while searching for a way to hide stories myself.
It is actually possible via tags.

.storybook/manager.ts
```ts
import { addons } from '@storybook/manager-api';

addons.setConfig({
  sidebar: {
    filters: {
      hidden: item => {
        return !item.tags?.includes('hidden');
      }
    }
  }
});
```

example.stories.ts
```ts
const meta = {
  title: 'Meta/Example',
  tags: ['autodocs', 'hidden']
};
```

If you annotate all stories whose title is prefixed by `Meta/`, the whole section actually disappears from Storybook's UI.

Reply:
Actually you can even match on the title:

`.storybook/manager.ts`

```ts
import { addons } from '@storybook/manager-api';

addons.setConfig({
  sidebar: {
    filters: {
      hidden: item => {
        return !item.title.startsWith('Meta/');
      }
    }
  }
});
```

This way you don't need to do any changes to your stories.